### ISSUE TITLE
CSS Literal String

### ISSUE THREAD

**Issue by little-wow** [creativecommons/communications#22]
Try as I might, I don't know how to figure out this code for emails, and Civi is still borking things.

**Comment by mattl** [comment]
Can you paste up your email template source on GitHub?

**Comment by little-wow** [comment]
https://github.com/creativecommons/communications/blob/master/newsletter-july.html
https://litmus.com/builder/documents/772541/edit

I used the template called "Email newsletter" from Litmus. I also did a bunch of in-GUI edits in Civi, which looked fine in all the test sends (mobile and desktop), saved file on my computer, and the GUI, but I just re-uploaded them into Litmus and that appears to have borked the template. I should have re-uploaded one more time into Litmus but figured it would be fine because it looked fine everywhere else.
